emission/tests/storageTests/TestSectionQueries.py

```diff
@@ -54,6 +76,50 @@ def testQuerySections(self):
             estt.TimeQuery("data.start_ts", 4, 6))
         self.assertEqual([entry.data for entry in ret_arr_list], ret_arr_time)
 
+    def testCleaned2InferredSectionList(self):
+
+        # Running the pipeline for the two user datasets
+        epi.run_intake_pipeline_for_user(self.testUUID1, skip_if_no_new_data = False)
+        epi.run_intake_pipeline_for_user(self.testUUID2, skip_if_no_new_data = False)
+
+        # Fetching the timeseries entries containing both raw data and analysis data after running intake pipeline
+        ts_agg = esta.TimeSeries.get_aggregate_time_series()
```

Comment:
for clarification: this doesn't actually fetch the user entries - it just fetches the appropriate access layer.